test/rekt/features/new_trigger_filters/filters.go

```diff
@@ -17,39 +17,49 @@ limitations under the License.
 package new_trigger_filters
 
 import (
+	"context"
 	"fmt"
 
+	"github.com/cloudevents/sdk-go/v2/event"
 	. "github.com/cloudevents/sdk-go/v2/test"
+	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
+	"knative.dev/eventing/test/rekt/resources/broker"
+	"knative.dev/eventing/test/rekt/resources/trigger"
 	"knative.dev/reconciler-test/pkg/eventshub"
 	. "knative.dev/reconciler-test/pkg/eventshub/assert"
 	"knative.dev/reconciler-test/pkg/feature"
 	"knative.dev/reconciler-test/pkg/manifest"
 	"knative.dev/reconciler-test/pkg/resources/service"
-
-	"github.com/cloudevents/sdk-go/v2/event"
-	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
-	"knative.dev/eventing/test/rekt/resources/broker"
-	"knative.dev/eventing/test/rekt/resources/trigger"
 )
 
 func createNewFiltersFeature(f *feature.Feature, eventContexts []CloudEventsContext, filters []eventingv1.SubscriptionsAPIFilter, filter eventingv1.TriggerFilter, installBroker InstallBrokerFunc) {
 	subscriberName := feature.MakeRandomK8sName("subscriber")
 	triggerName := feature.MakeRandomK8sName("trigger")
-	brokerName := installBroker(f)
+	brokerName := feature.MakeRandomK8sName("trigger")
```

Comment:
nit: naming the broker with prefix `broker` might be a bit clearer
```suggestion
	brokerName := feature.MakeRandomK8sName("broker")
```

Reply:
Applied the suggested change. Thanks!